### ISSUE TITLE
missing agency name 

### ISSUE THREAD

**Issue by gbinal** [18F/pulse#553]
https://staging.pulse.cio.gov/accessibility/agencies/

![screen shot 2016-09-16 at 12 35 31 pm](https://cloud.githubusercontent.com/assets/633088/18593585/16a8bbd0-7c0a-11e6-83a1-e6b8126d122c.png)

**Comment by gbinal** [comment]
The missing agency field is missing from the underlying dataset (last record on [here](https://github.com/18F/pulse/blob/master/static/data/tables/accessibility/agencies.json)).  I've compared the agencies listed on the [agencies tab](https://staging.pulse.cio.gov/accessibility/agencies/) with the [list of domains](https://github.com/18F/pulse/blob/master/static/data/tables/accessibility/domains.json) and not been able  find an agency represented on one but not the other.  

I suspect that this is being caused in the scripting process though won't be able to know for sure until we finish going through the process holistically.  #557 will remove the row temporarily until we can update the data with scans done that we've affirmed don't recreate this issue.

**Comment by gbinal** [comment]
This is resolved.
